contracts/evm/test/xcall/CallService.t.sol

```diff
@@ -385,6 +407,27 @@ contract CallServiceTest is Test {
         callService.executeCall(1, data);
     }
 
+    function testExecuteCallPersistentMessage() public {
```

Comment:
Test should be execute and dapp reverts, then be able to execute again and succeed.